AUTHORS.rst

```diff
@@ -15,3 +15,4 @@ We manually track this since commit author *does not* equal code author.
 * Markus Unterwaditzer <markus@unterwaditzer.net>
 * Paweł Fertyk <pfertyk@openmailbox.org>
 * Thomas Glanzmann <thomas@glanzmann.de>
+* Swati Garg <swati4star@gmail.com>
```

Comment:
This list is in alphabetical order (I'll add a clarification this time, since it's not at all obvious, really my bad).